ci/test/dot

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -ex
+
+source ./common
+init_test_repo
+cd $test_repo_path
+
+#####
+# Logic
+# The actual output is compared to expected output
+# The actual output is presorted
+# The expected output is sorted
+# Sorting is done as output is sometimes unpredictable
+#####
+
+
+## Testing default file feature
+git clone https://github.com/cplee/github-actions-demo.git
+cd github-actions-demo
+
+export PHONY_SECRET=foo
+
+popper dot > out
+actual_output=`tail -n+2 out | head -n-1 | cut -f2 | sort`
+
+expected_output="branch_filter -> lint;
+branch_filter -> test;
+deploy -> branch_filter;
+lint -> install;
+test -> install;"
```

Comment:
deploy is the last action as shown [here](https://github.com/cplee/github-actions-demo/blob/master/.github/main.workflow)

Reply:
The `expected_output` is actual a list of sorted edges. Sorting is needed as graph nodes on the same level may be executed in arbitrary order. Eg. `lint -> install` and `test -> install` were appearing out of order. So sort was used to rectify this, so that comparison is done properly.
But I think `expected_output` is misleading, will rename it to something else.